pkg/kubectl/cmd/run.go

```diff
@@ -254,11 +256,21 @@ func Run(f *cmdutil.Factory, cmdIn io.Reader, cmdOut, cmdErr io.Writer, cmd *cob
 		if err != nil {
 			return err
 		}
-		err = handleAttachPod(f, client, attachablePod, opts, quiet)
+		err = handleAttachPod(f, client, attachablePod.Namespace, attachablePod.Name, opts, quiet)
 		if err != nil {
```

Comment:
nit: Wonder, we are using quiet here but we are reading that in the first line of this function?